utils.js

```diff
@@ -399,6 +399,20 @@ function formatReadReceipt(event) {
   };
 }
 
+function formatRead(event) {
+  return {
+    tids: event.tids,
```

Comment:
If you look at the other objects, you will see they are all camelcased (`folderInfo` instead of folder_info).  We try to do some work to gain consistency as well by combining together various properties.  I would probably suggest something like the following:
```js
{
    threadID: event.chat_ids[0] || event.thread_fbids[0], // ID of the thread you read
    timestamp: event.timestamp,
    userID: event.realtime_viewer_fbid, // Your ID
    type: 'read'
}
```
Arguably the userID could be removed if we know that it will always be your id.  If `mark_as_read` is always true we shouldn't include it.  I doubt that `folder_info` will be useful to anyone.  As a note, we treat a user's Facebook userID as their threadID in a single-user chat with them (that explains the `threadID` property).
@bsansouci anything I'm missing?